### DISCUSSION TITLE
Running collections with data files

### DISCUSSION THREAD

**Discussion by piotr-szyk** [thunderclient/thunder-client-support#560] [Q&A]
Hi Community,

Could you please advise if Thunder Client has the capability to run API collections with data files like Postman(https://learning.postman.com/docs/running-collections/working-with-data-files/)?

I would like to be able to iterate through values taken from external data file (csv or json). 

Example:
GET myapi/{{user}}

Data file (csv example):
user
Adam
Monica
Ranga

The expected outcome would be the following API requests:
GET myapi/Adam
GET myapi/Monica
GET myapi/Ranga

I hope this makes sense. Please advise. 

Thank you very much in advance.

technoorange514

**Comment by rangav** [comment]
Thanks @technoorange514 for the feedback.

Its already in roadmap. you can add your use-case here
https://github.com/rangav/thunder-client-support/issues/95
